cmd/multiFaExtract/testdata/chr1.60.200.expected.noGap.fa

```diff
@@ -0,0 +1,9 @@
+>hg38
+GGAGGCGCGGGGAGGCCCCTTACCCGCCCCACTGTCCTGCCCGCCAATGA
```

Comment:
This is giving the output I expected from the code, but for your work I thought you needed them to stay aligned?

Reply:
In some cases we may not, for example if you wanted to take individual ungapped sequences to design primers or order synthesis and no longer cared about the alignment but wanted ungapped sequence. Maybe a niche case but the default behavior retains alignment and the usage clearly warns the user